src/app/vm/shared/vm.model.ts

```diff
@@ -66,5 +67,8 @@ export interface VirtualMachine extends BaseModel, Taggable {
   keypair: string;
   password: string;
   passwordenabled: boolean;
-  instanceGroup: InstanceGroup;
 }
+
+export const getInstanceGroup = (vm: VirtualMachine) => {
+  return vm.tags.find(tag => tag.key === VirtualMachineTagKeys.group);
```

Comment:
We have an InstanceGroup class, use it or change it to the interface. Coz you return Tag interface and this is strange.